pkg/detectors/datadogtoken/datadogtoken_integration_test.go

```diff
@@ -54,6 +54,10 @@ func TestDatadogToken_FromChunk(t *testing.T) {
 					ExtraData: map[string]string{
 						"Type": "Application+APIKey",
 					},
+					AnalysisInfo: map[string]string{
```

Comment:
The analysis info will require the endpoint as well.

Reply:
Also we need to update the analyzer a bit to take the domain as input. We can connect about that and I can explain you how we can do that.